code/modules/vending/mail.dm

```diff
@@ -0,0 +1,308 @@
+#define STATE_SORTING "sorting"
+#define STATE_IDLE "idle"
+#define STATE_YES "yes"
+#define STATE_NO "no"
+
+/obj/machinery/mailsorter
+	name = "mail sorter"
+	desc = "A large mail sorting unit. Sorting mail since 1987!"
+	icon = 'icons/obj/machines/mailsorter.dmi'
+	icon_state = "mailsorter"
+	layer = BELOW_OBJ_LAYER
+	density = TRUE
+	max_integrity = 300
+	integrity_failure = 0.33
+	var/light_mask = "mailsorter-light-mask"
+	var/panel_type = "panel"
+
+	circuit = /obj/item/circuitboard/machine/mailsorter
+
+	/// How much mail can the mail sorter store.
+	var/mail_capacity = 100
```

Comment:
If this never gets changed (like through upgrading the parts) then it should be a define instead